api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/DataPoint.java

```diff
@@ -18,14 +18,14 @@
 
 import javax.xml.bind.annotation.XmlRootElement;
 
-import com.wordnik.swagger.annotations.ApiClass;
-import com.wordnik.swagger.annotations.ApiProperty;
+import com.wordnik.swagger.annotations.ApiModel;
+import com.wordnik.swagger.annotations.ApiModelProperty;
 
```

Comment:
Swagger java-version does not follow the specification version numbering. But yes, there's lots of changes between Java 1.2 -> 1.3, but relatively little from 1.3 -> 1.5 (1.5 isn't ready yet, it's milestone at the moment).